### DISCUSSION TITLE
Osu! Lazer .Appimage won't open

### DISCUSSION THREAD

**Discussion by jelyfzh** [ppy/osu#23060] [Q&A]
### Type

Crash to desktop

### Bug description

I can't launch the .appimage at all. Just refuses to open. I made it executable in the properties but still no dice. I can see it breifly start up for a second or two in task manager but then it just immedietly quits afterwards. Have updated my OS, rebooted several times. For reference I am on a Dell Chromebook 11 3120 with 2 GB ram and a hacked bios running GalliumOS. It's ubuntu-ish based so i don't know what the problem is.

### Version

the latest one at the time of me submitting this

### Logs

i don't have any

**Comment by bdach** [comment]
Please check output of `dmesg` for relevant errors.

**Comment by Bruno5430** [comment]
are you able to run any other `.appimage`s? Most common cause of `.appimage`s failing to launch is because distros don't ship with `libfuse2` by default, but you can [install it back](https://docs.appimage.org/user-guide/troubleshooting/fuse.html#setting-up-fuse-2-x-on-ubuntu-pre-22-04-debian-and-their-derivatives)
(Also, GalliumOS is EOL and no longer supported by it's mantainers, see the [wiki](https://wiki.galliumos.org/Welcome_to_the_GalliumOS_Wiki))

**Reply by Xaverosu** [reply]
THANK YOU!!!! Worked for me! <3

**Reply by cymuii** [reply]
This worked for me too! You are a life saver!!!
Just curious I can run lazer previously without being aware i need to install this.
Do you know why all in a sudden i'd need this package to run osu again?

**Comment by Schweeeeeeeeeeeeeeee** [comment]
Run the AppImage in a terminal, that usually shows me what's wrong with it.

**Reply by jelyfzh** [reply]
when i run it in terminal it gives me this:

`robotguy600000@galliumbook:~/Downloads$ ./osu.AppImage 
touch: cannot touch '/home/robotguy600000/.local/share/icons/hicolor/.xdg-icon-resource-dummy': No such file or directory
Unhandled exception. System.DllNotFoundException: Unable to load shared library 'SDL2' or one of its dependencies. In order to help diagnose loading problems, consider setting the LD_DEBUG environment variable: libSDL2: cannot open shared object file: No such file or directory
   at SDL2.SDL.INTERNAL_SDL_SetHint(Byte* name, Byte* value)
   at SDL2.SDL.SDL_SetHint(String name, String value)
   at osu.Framework.Platform.Linux.LinuxGameHost.SetupForRun()
   at osu.Framework.Platform.GameHost.Run(Game game)
   at osu.Desktop.Program.Main(String[] args) in /var/lib/buildkite-agent/builds/debian-gnu-linux-vm-1/ppy/osu/osu.Desktop/Program.cs:line 120
Aborted (core dumped)
`

*edit i installed the sdl and now i just get this

`robotguy600000@galliumbook:~/Documents$ ./osu.AppImage 
touch: cannot touch '/home/robotguy600000/.local/share/icons/hicolor/.xdg-icon-resource-dummy': No such file or directory
Unhandled exception. System.NullReferenceException: Object reference not set to an instance of an object.
   at osu.Framework.Input.Handlers.Tablet.OpenTabletDriverHandler.Initialize(GameHost host)
   at osu.Framework.Platform.GameHost.initialiseInputHandlers()
   at osu.Framework.Platform.GameHost.Run(Game game)
   at osu.Desktop.Program.Main(String[] args) in /var/lib/buildkite-agent/builds/debian-gnu-linux-vm-1/ppy/osu/osu.Desktop/Program.cs:line 120
Aborted (core dumped)
`

**Reply by Bruno5430** [reply]
please try running any other appimage as requested in https://github.com/ppy/osu/discussions/23060#discussioncomment-5567548 to see if the problem persist

**Reply by jelyfzh** [reply]
the appimagelauncher appimage launched and installed just fine
